Default MPICH instructions

milljm opened the pull request:
Add the necessary bit to ensure users receive our vetted MPICH variant conda stack.

Add some information about when/where to seek help for an optimized environment.

Closes #28193

Commit:
Default MPICH instructions

Add the necessary bit to ensure users receive our vetted MPICH variant
conda stack.

Add some information about when/where to seek help for an optimized
environment.

Closes #28193

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2345595/) on a7cbd76 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/28195/site)

This comment will be updated on new commits.

Commit:
Optionally include index.md

For those applications that are not including the entire getting
started documentation stack.
